libvirt/qemu_agent.go

```diff
@@ -24,75 +26,98 @@ type QemuAgentInterfaceIpAddress struct {
 	Prefix  uint   `json:"prefix"`
 }
 
+type libvirtIfacesLst []libvirt.DomainInterface
```

Comment:
There's no real benefit in doing this. Simply writing `[]libvirt.DomainInterface` makes it more understandable IMHO.